cli/bpmetadata/cmd.go

```diff
@@ -203,3 +253,49 @@ func createContent(bpPath string, rootPath string, readmeContent []byte) Bluepri
 
 	return content
 }
+
+func writeMetadata(obj *BlueprintMetadata) error {
```

Comment:
Let's add some test cases so we can visualize the rendered yaml.

Reply:
Will do. b/254323527